server/app/lib/actions/fusor/deployment/open_shift/install_ose.rb

```diff
@@ -10,6 +10,8 @@
 # have received a copy of GPLv2 along with this software; if not, see
 # http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
 
+require "#{Fusor::Engine.root}/lib/modules/ose-installer/launch.rb"
```

Comment:
It's a quick and dirty way to get the module imported. I had trouble with the namespace but did not want to stop making progress. I've created a card https://trello.com/c/qYu1dgnj to clean up the code at a later date.